microsite/src/pages/plugins/plugins.module.scss

```diff
@@ -28,10 +28,43 @@
   :global(.pluginsContainer) {
     gap: 1rem;
     display: grid;
-    grid-template-columns: repeat(auto-fit, minmax(250px, 1fr));
+    grid-template-columns: repeat(auto-fit, minmax(250px, 300px));
```

Comment:
Just going to mention that in the last PR I wasn't sure about this - worry about the impact of a hardcoded size like this. @benjdlambert any thoughts on this?

Reply:
I'm also not fan of fixed values in styles but not to much expertise there.

The reason why I fix it is because if you use 1fr and you filter card plugins (e.g you have just 1 or 2) the cards width will increase a lot

Any suggestion for avoid taht behaviour without fixed width?

PD: I made some fast tests resizing the window and also in device view and everything ok